testing/e2e/config/config.go

```diff
@@ -182,17 +182,17 @@ func defaultValidators() NodeSet {
 		Nodes: []Node{
 			{
 				ElType:   "nethermind",
-				Replicas: 1,
+				Replicas: 0, // nethermind cannot keep up with deposits checks
 				KZGImpl:  "crate-crypto/go-kzg-4844",
 			},
 			{
 				ElType:   "geth",
-				Replicas: 2, //nolint:mnd // bet.
+				Replicas: 2, //nolint:mnd // we want two replicas here
 				KZGImpl:  "crate-crypto/go-kzg-4844",
 			},
 			{
 				ElType:   "reth",
```

Comment:
debugging changes. It seems that:

- Dropping nethermind make e2e tests pass (nethermind seems to miss some blocks containing deposits)
- A network made of nethermind only nodes cannot even start

If confirmed, we may have to drop nethermind from e2e tests (simiarly to what we do for Besu) and re-evaluate later on if/how reactivate them

Reply:
Filed an issue https://github.com/berachain/beacon-kit/issues/2177 to track Nethermind issue (this happens on main too)